src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/CostsService.java

```diff
@@ -15,10 +15,13 @@ public class CostsService {
 
     private static final String UPDATE_COST_DESCRIPTION = "Update Overseas Entity fee";
 
+    private static final String REMOVE_COST_DESCRIPTION = "Remove Overseas Entity fee";
+
     private static final String PAYMENT_ACCOUNT = "data-maintenance";
     private static final String RESOURCE_KIND = "overseas-entity";
     private static final String REGISTER_PRODUCT_TYPE = "register-overseas-entity";
     private static final String UPDATE_PRODUCT_TYPE = "update-overseas-entity";
+    private static final String REMOVE_PRODUCT_TYPE = "remove-overseas-entity";
```

Comment:
Not sure who (or what) uses the product type. @markpit ?

I've defined this value but no idea if it's correct or needs communicating to someone.

Also, PG has informed me that product code is 16303 for removal. Not sure if that needs to feature in our code (web or API) somewhere?

Reply:
i think it is used in the services downstream of the api, after the JSON is sent to kafka

Reply:
I now think this definition is ok and I can see where the 16303 code needs defining. See [this](https://companieshouse.atlassian.net/browse/UAR-890) ticket. I'll create a new PR for the `payment-reconciliation-consumer` repo to tie these things together.

Reply:
PR is [here](https://github.com/companieshouse/payment-reconciliation-consumer/pull/50).